src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -94,6 +94,11 @@ def load_arguments(self, _):
     with self.argument_context('webapp update') as c:
         c.argument('client_affinity_enabled', help="Enables sending session affinity cookies.", arg_type=get_three_state_flag(return_label=True))
         c.argument('https_only', help="Redirect all traffic made to an app using HTTP to HTTPS.", arg_type=get_three_state_flag(return_label=True))
+        c.argument('force_dns_registration', help="If true, web app hostname is force registered with DNS", arg_type=get_three_state_flag(return_label=True))
```

Comment:
those are command arguments which are already exposed through the update. Because I updated the wiring up mechanism for slot support which involves less auto-command support, hence I have to manually author the command argument here.